test/translate_c.zig

```diff
@@ -328,7 +328,7 @@ pub fn addCases(cases: *tests.TranslateCContext) void {
         \\pub const LIGHTGRAY = @import("std").mem.zeroInit(CLITERAL(Color), .{ @as(c_int, 200), @as(c_int, 200), @as(c_int, 200), @as(c_int, 255) });
         ,
         \\pub const struct_boom_t = extern struct {
-        \\    i1: c_int,
+        \\    @"i1": c_int,
```

Comment:
This is incorrect; the previous test case was correct. Same for `null` - zig allows `i1` and `null` to be field names without using `@""` syntax.